### PR TITLE
feat(Aurora): add Aurora Layer2 token list

### PR DESCRIPTION
**Pull request by developerfred**
### Default/Bridge Token List
- [maininet](https://mainnet.aurora.dev/tokens )

### Dexs Tokens List
- [Trisolaris](https://github.com/trisolaris-labs/tokens/blob/master/lists/1313161554/list.json)
- [Wannaswap](https://github.com/wannaswap/tokens/tree/master/blockchains/aurora/assets)

### CODE REVIEW

**Comment by UncleBill** [comment]
Hi, @developerfred, thanks for you contribution. 
I would to know where the token list`src/erc20/aurora.json` comes from？I wanna do a double check.

**Comment by developerfred** [comment]
- [maininet](https://mainnet.aurora.dev/tokens )
- [testmet](https://testnet.aurora.dev/tokens )
- [betanet](https://betanet.aurora.dev/tokens )

**Comment by UncleBill** [comment]
@developerfred Thanks, I have confirmed the list now.

**Review by UncleBill — DISMISSED** [review]

**Comment by UncleBill** [comment]
Hi, @developerfred , you need to build out token-list. I already did it in
https://github.com/DimensionDev/Mask-Token-List/tree/feat/AuroraTokenList

you can sync your branch with it.

**Comment by developerfred** [comment]
> `feat`/AuroraTokenList

I will do this.

**Comment by developerfred** [comment]
@UncleBill done

**Review by UncleBill — APPROVED** [review]

File: src/erc20/aurora.json [hunk 1]
```diff
@@ -0,0 +1,440 @@
+[
+  {
+    "chainId": 1313161554,
+    "address": "0x4e834cdcc911605227eedddb89fad336ab9dc00a",
+    "symbol": "AAVE",
+    "name": "Aave Token",
+    "decimals": 18,
+    "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/aave.svg",
+    "tags": ["ethereum"]
```
Comment (guanbinrui) [inline src/erc20/aurora.json:9]: Unused field? Please remove it.

Comment (developerfred) [reply src/erc20/aurora.json:9]: OK!

**Review by UncleBill — DISMISSED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1429 @@
+{
```
Comment (guanbinrui) [inline package-lock.json:1]: Delete this file.

**Review by guanbinrui — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (developerfred) [reply package-lock.json]: done

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**